src/pages/User/components/PullRequests/ShareButtons.js

```diff
@@ -9,9 +9,7 @@ const ShareButtons = ({ username, pullRequestCount }) => (
         target="_blank"
         rel="noopener noreferrer"
         className="bg-blue text-white rounded px-2 py-1 pointer text-white no-underline text-sm"
-        href={`https://twitter.com/intent/tweet?text=My progress on hacktoberfest ${pullRequestCount} / ${pullRequestAmount}&url=${
-          process.env.REACT_APP_HOSTNAME
-        }/username/${username}&hashtags=hacktoberfest, hacktoberfestchecker`}
+        href={`https://twitter.com/intent/tweet?text=My progress on hacktoberfest ${pullRequestCount} / ${pullRequestAmount}&url=${process.env.REACT_APP_HOSTNAME}/username/${username}&hashtags=hacktoberfest, hacktoberfestchecker`}
```

Comment:
change `hacktoberfest` to `frogtoberfest`. Also we should create a new file `config.js` that will export variables from the `env` file. Let's meet if you have confusion with the second part.

Reply:
@aviskarkc10  Dai.  Regarding the second part, why should we make a separate config file? Every time we need to use the env variables in our js file we need to make an import to that file. Won't it be hectic to make an import, then use the variable rather than directly use the variable defined in .env file without using any import which React has given?